readthedocs/config/config.py

```diff
@@ -622,7 +625,10 @@ def conda(self):
     @property
     def build(self):
         """The docker image used by the builders."""
-        return Build(**self._config['build'])
+        return Build(
+            apt_packages=[],
```

Comment:
This could use a comment. I don't really understand why it's here.

Reply:
We don't have "defaults", so we need to pass all values when building a `Build` object.

Reply:
I wanted to refactor this to allow to set some defaults in the class, maybe that's an easy fix